addons/dogtags/functions/fnc_canTakeDogtag.sqf

```diff
@@ -20,4 +20,7 @@ params ["_player", "_target"];
 
 if (isNull _target) exitWith {false};
 
+// check if disabled for faction
+if ([GVAR(disabledFactions) getVariable faction _target] param [0, false]) exitWith {false};
```

Comment:
Only works now because we create the CBA namespace on mission start by calling the disableFactionDogtags. If we ever change that, this will break due to undefined variable.